system/dialogcolors.xml

```diff
@@ -53,73 +51,96 @@
 	<color name="Fuzzy Wuzzy">FFCC6666</color>
 	<color name="Gold">FFE7C697</color>
 	<color name="Goldenrod">FFFCD975</color>
+	<color name="Granite Gray">ff616161</color>
 	<color name="Granny Smith Apple">FFA8E4A0</color>
-	<color name="Green">FF1CAC78</color>
+	<color name="Gray Arsenic">ff424242</color>
+	<color name="Gray Cultured">fff5f5f5</color>
+	<color name="Gray Lotion">fffafafa</color>
+	<color name="Gray XII">ffbdbdbd</color>
 	<color name="Green Blue">FF1164B4</color>
 	<color name="Green Yellow">FFF0E891</color>
+	<color name="Green">FF1CAC78</color>
+	<color name="Hot Cocoa">FF8F482F</color>
 	<color name="Hot Magenta">FFFF1DCE</color>
 	<color name="Inchworm">FFB2EC5D</color>
 	<color name="Indigo">FF5D76CB</color>
+	<color name="Iridescent Indigo">FF3C32CD</color>
 	<color name="Jazzberry Jam">FFCA3767</color>
 	<color name="Jungle Green">FF3BB08F</color>
 	<color name="Laser Lemon">FFFEFE22</color>
+	<color name="Lavender Silk">FF6B4D82</color>
 	<color name="Lavender">FFFCB4D5</color>
+	<color name="Leafy Canopy">FF94DDCB</color>
 	<color name="Lemon Yellow">FFFFF44F</color>
+	<color name="Liquid Gold">FFFFD966</color>
 	<color name="Macaroni and Cheese">FFFFBD88</color>
 	<color name="Magenta">FFF664AF</color>
 	<color name="Magic Mint">FFAAF0D1</color>
 	<color name="Mahogany">FFCD4A4C</color>
 	<color name="Maize">FFEDD19C</color>
 	<color name="Manatee">FF979AAA</color>
+	<color name="Mango Purée">FFFF6137</color>
 	<color name="Mango Tango">FFFF8243</color>
 	<color name="Maroon">FFC8385A</color>
 	<color name="Mauvelous">FFEF98AA</color>
 	<color name="Melon">FFFDBCB4</color>
 	<color name="Midnight Blue">FF1A4876</color>
+	<color name="Moonlit Pond">FF4F2CD0</color>
 	<color name="Mountain Meadow">FF30BA8F</color>
 	<color name="Mulberry">FFC54B8C</color>
 	<color name="Navy Blue">FF1974D2</color>
 	<color name="Neon Carrot">FFFFA343</color>
+	<color name="Ocean Foam">FF62C9D3</color>
 	<color name="Olive Green">FFBAB86C</color>
-	<color name="Orange">FFFF7538</color>
+	<color name="Orange Peel">FFFF8021</color>
 	<color name="Orange Red">FFFF2B2B</color>
 	<color name="Orange Yellow">FFF8D568</color>
+	<color name="Orange">FFFF7538</color>
```

Comment:
How come `"Orange" `comes after `"Orange whatever"`?

Reply:
Sorting is lexicographic. For english letters this means alphabetical order, but for non-letters sorting is done by ASCII value.